app/views/tags/aggregatedMetrics.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+* Copyright 2016 LinkedIn Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*@
+
+@(result: models.AppResult)
+@import com.linkedin.drelephant.util.Utils;
+
+
+@if(!result.jobType.equals("Spark")) {
+  <p class="list-group-item-heading">
+<table>
+  <td>
+<td title="The consumed resources in GB Hours" class="metrics-dashboard" width="150px"><img src='@routes.Assets.at("images/usedmemory.png")' class="metrics-icons" alt='used resources'/> @Utils.getDurationInGBHours(result.resourceUsage)</td>
```

Comment:
I'd prefer using unicode characters rather than loading images. While listing 50 jobs in the dashboard, we will have 200 images to load.

**Replacement Suggestion:**
Res Usage: &#9775;
Wasted Res Usage: &#8855;
Runtime: &#9719;
Wait Time: &#10710;